schemas/public/kfddistribution-kfd-v1alpha2.json

```diff
@@ -714,6 +714,10 @@
         "deadManSwitchWebhookUrl": {
           "type": "string"
         },
+        "defaultRules":  {
+          "type": "string",
+          "enum": ["enabled", "disabled"]
```

Comment:
Same as above, we should change to installDefaultRules and a boolean value

Reply:
I got an error when on "reducer" when try to put a boolean